Switch the default virtual memory handling method to TLB reprogramming by the debugger.

Signed-off-by: Chris Smith <chris.smith@st.com>

Index: gdb-6.8/gdb/linux-awareness-sh4.c
===================================================================
--- gdb-6.8.orig/gdb/linux-awareness-sh4.c	2009-03-10 10:07:46.000000000 +0000
+++ gdb-6.8/gdb/linux-awareness-sh4.c	2009-03-10 10:07:57.000000000 +0000
@@ -30,7 +30,7 @@
 /* TLB cache */
 static unsigned int current_tlb_config_virt;
 static unsigned int current_tlb_config_phys;
-static int force_tlb_reprogramming;
+static int force_tlb_reprogramming = 1;
 
 static int try_cached_translation(CORE_ADDR *addr)
 {
